website/snippets/connect-starburst-trino/schema-db-fields.md

```diff
@@ -1,3 +1,3 @@
 [comment: For context, the section title used for this snippet is "Schemas and databases" ]: # 
 
-When selecting the database (catalog) and the schema, make sure the user has read and write access to both the provided database (catalog) and schema. This selection does not limit your ability to query the catalog. Instead, they serve as the default location for where tables and views are materialized. This _default_ can be changed later from within your dbt project.
\ No newline at end of file
+When selecting the catalog and the schema, make sure the user has read and write access to both. This selection does not limit your ability to query the catalog. Instead, they serve as the default location for where tables and views are materialized. In addition, the Trino connector used in the catalog must support creating tables and views in the catalog. This _default_ can be changed later from within your dbt project.
```

Comment:
Support for creating tables and views are not a hard requirement here. If a connector doesn't support view creation, it doesn't mean that a user cannot create tables.